style.css

```diff
@@ -61,43 +74,276 @@ body {
   color: #505f79;
 }
 
-.section2 {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  height: 100vh;
-  background-image: url("./assits/header-shapes mobile.png");
-  background-color: #fff;
-  background-repeat: no-repeat;
-  background-size: 100%;
-}
-
 .welcome {
   font-family: Poppins, sans-serif;
-  font-size: 40px;
+  font-size: 68px;
   font-weight: 700;
-  line-height: 52px;
+  line-height: 90px;
   text-align: left;
   color: #172b4d;
   margin-left: 24px;
 }
 
 .about_me {
   font-family: Poppins, sans-serif;
-  font-size: 16px;
+  font-size: 20px;
   font-weight: 400;
-  line-height: 24px;
+  line-height: 28px;
   text-align: left;
   color: #344563;
   margin-left: 24px;
+  width: 70%;
 }
 
 .connect_me a {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 24px;
   color: #7f8cff;
   text-decoration: none;
-  font-size: 16px;
-  font-weight: 500;
   line-height: 24px;
   margin-left: 24px;
 }
+
+.cards {
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
```

Comment:
- [ ] Kindly use grid instead of flex because it is a requirements. :slightly_smiling_face: 

Check below :point_down:  and also check this [link](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) for more information:
> In order to lay out the elements in the "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.